compiler/rustc_codegen_llvm/src/llvm_util.rs

```diff
@@ -237,16 +231,7 @@ pub fn target_features(sess: &Session, allow_unstable: bool) -> Vec<Symbol> {
             true
         })
         .map(|feature| Symbol::intern(feature))
-        .collect();
-
-    // LLVM 14 changed the ABI for i128 arguments to __float/__fix builtins on Win64
-    // (see https://reviews.llvm.org/D110413). This unstable target feature is intended for use
-    // by compiler-builtins, to export the builtins with the expected, LLVM-version-dependent ABI.
-    // The target feature can be dropped once we no longer support older LLVM versions.
-    if sess.is_nightly_build() && get_version() >= (14, 0, 0) {
-        features.push(Symbol::intern("llvm14-builtins-abi"));
-    }
```

Comment:
So this is somewhat unfortunate in that this introduces a constraint on the version of compiler-builtins that works with the new version of LLVM due to how this feature was set up (rather than enabling the behaviour for old LLVM versions it enables the behaviour for the new LLVM versions). I wonder if it would make sense to keep setting this feature for a little longer…? Maybe I’m overthinking this and nobody is really depending on the crate except for rustc?

Reply:
The risk would be using an older version of `compiler_builtins` with a nightly where we've removed this feature -- but I think that's kind of par for the course with nightly crates. It looks like most of the [reverse dependencies](https://crates.io/crates/compiler_builtins/reverse_dependencies) are just for `rustc-std-workspace` hacking, but I'm not sure.